### ISSUE TITLE
"Crack" in brush on gm_construct

### ISSUE THREAD

**Issue by imp0se** [Facepunch/garrysmod-issues#2237]
![image](https://cloud.githubusercontent.com/assets/8015170/9837798/5b5f50ca-5a25-11e5-9f68-577144d5f442.png)

There's a small crack in this brush on gm_construct. Can be easily fixed by moving around some of the vertices by ~0.5HU or so. This crack gets thinner further on into the brush, and eventually, it slightly clips into another beside it.

**Comment by Kefta** [comment]
No one is going to recompile gm_construct just to fix this (among the other lighting/clipping issues). Maybe if we ever get a new construct map, this won't be an issue.

**Comment by immibis** [comment]
@Kefta Should be a trivial fix if someone has the original VMF files, shouldn't it?

**Comment by Kefta** [comment]
@immibis Yes, but then you have to deal with map versioning conflicts, and are you going to make a v2 just to fix a brush crack?

**Comment by immibis** [comment]
@Kefta I thought maps distributed with the game could be updated easily. You won't get e.g. clients with the new map playing on servers with the old map, because the client version has to match the server version anyway.

**Comment by Grocel** [comment]
Map versioning conflicts is only a thing in non official maps. I guess you would be surprised how often Valve updates their official maps.
